Automattic/jetpack
# Jetpack: Have webpack embed social-logos.woff2 in CSS

## Proposed changes:

Having to manually construct the data URI caused some breakage recently. Let's have Webpack do the work instead.

### Other information:


## Jetpack product discussion


p9dueE-7DU-p2#comment-9905

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Built `_inc/social-logos/social-logos.min.css` looks ok?
* Check that the stuff using this CSS rule still works.
  * Use the built artifact, not a local build, to ensure everything is in the artifact correctly.
  * Check all supported browsers to ensure nothing has a problem with the mime type in the data URI being "font/woff2" rather than "application/octet-stream" like it was before.
